pkg/kubelet/dockershim/network/kubenet/kubenet_linux_test.go

```diff
@@ -342,4 +342,57 @@ func TestGetRoutesConifg(t *testing.T) {
 	}
 }
 
+func TestGetRangesConfig(t *testing.T) {
+	for _, test := range []struct {
+		cidrs    []string
+		gateways []string
+		ranges   string
+	}{
+		{
+			cidrs:    []string{"10.0.0.1/24"},
```

Comment:
testing with CIDR strings like that is just wrong... I'd squash the last two commits together. I don't think there's any reason to add the test in a broken form first, and then fix it